Android/Java/app/build.gradle

```diff
@@ -1,11 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-    namespace "com.ironsource.ironsourcesdkdemo"
+    namespace "com.levelplay.levelplaydemo"
```

Comment:
is this the requirement? do we have a ticket with the product definition of the new namespace?